src/commands.ts

```diff
@@ -18,12 +18,17 @@ import {
   promptForFavoriteColorName
 } from './inputs';
 
+import { resetLiveSharePreviousColors } from './live-share/liveshare-commands';
+
 export async function resetColorsHandler() {
   const colorCustomizations = deletePeacocksColorCustomizations();
   State.recentColor = '';
   const newColorCustomizations = isObjectEmpty(colorCustomizations)
     ? undefined
     : colorCustomizations;
+
+  resetLiveSharePreviousColors();
```

Comment:
Tried to keep the Live Share separated, this is one of the `2` functions that are added to the `Peacock` code.